pom.xml

```diff
@@ -520,20 +521,16 @@
 					<plugins>
 						<plugin>
 							<groupId>org.apache.maven.plugins</groupId>
-							<artifactId>${maven-compiler-plugin.version}</artifactId>
-							<version>3.10.1</version>
+							<artifactId>maven-compiler-plugin</artifactId>
+							<version>${maven-compiler-plugin.version}</version>
```

Comment:
This has been fixed in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1775